donate/settings.py

```diff
@@ -36,9 +37,11 @@ class DevConfig(Config):
 
     ENV = 'dev'
     DEBUG = True
-    DB_NAME = 'dev.db'
-    DB_PATH = os.path.join(Config.PROJECT_ROOT, DB_NAME)
-    SQLALCHEMY_DATABASE_URI = 'sqlite:////{0}'.format(DB_PATH)
+    # DB_NAME = 'dev.db'
+    # DB_PATH = os.path.join(Config.PROJECT_ROOT, DB_NAME)
+    SQLALCHEMY_DATABASE_URI = os.environ.get('TEST_DATABASE_URI',
+                                             'postresql://localhost/example')
+    # SQLALCHEMY_DATABASE_URI = 'sqlite:////{0}'.format(DB_PATH)
```

Comment:
Is there a reason to switch the dev config to use postgresql instead of sqlite?  I think this will make it harder for people to get setup to contribute to this project.

Reply:
The dialect differences can cause annoying issues.  There's an argument for something like a personal setup which uses sqlite, a test setup which uses sqlite (as-is) dev which is like developing in production where everything gets merged and tested first, then production.  

I think we can assume the test db is available and refresh is with production weekly or something..but in reality i dont think resolving all the above matters too much right now.  If we get 3 people to get comfortable with setting up everything including percona we'll be in good shape.